x-pack/plugin/sql/src/test/java/org/elasticsearch/xpack/sql/planner/QueryFolderTests.java

```diff
@@ -33,31 +32,28 @@
 
 public class QueryFolderTests extends ESTestCase {
 
-    private static SqlParser parser;
-    private static Analyzer analyzer;
-    private static Optimizer optimizer;
-    private static Planner planner;
+    private SqlParser parser;
+    private Analyzer analyzer;
+    private Optimizer optimizer;
+    private Planner planner;
+    private IndexResolution resolution;
 
-    @BeforeClass
-    public static void init() {
+    @Before
```

Comment:
What's the reason of not initialising those once?

Reply:
statics are tricky since they stick around; further more not all of them were cleaned.
Considering the classes are fairly light and don't have any external dependencies, instantiating them per test is not a big deal.